programs/drift/src/controller/orders.rs

```diff
@@ -398,6 +398,37 @@ pub fn place_perp_order(
     Ok(())
 }
 
+pub fn place_swift_perp_order(
```

Comment:
think it might be clearer if we delete this and just do the expected order id in the place_swift_taker_order fn